src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -8849,7 +8772,7 @@ unsafe static R Create()
 
         // Breaking change in C#11: A ref to ref struct argument is considered
         // an unscoped reference when passed to an __arglist.
-        [Theory]
+        [Theory(Skip = "https://github.com/dotnet/roslyn/issues/64308")]
```

Comment:
>Skip = "https://github.com/dotnet/roslyn/issues/64308" [](http://example.com/codeflow?start=16&length=54)

Consider unskipping the test and including the actual errors reported below, along with a link to the bug.

Reply:
Is there still a breaking change here with `__arglist`? If not, please remove [https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20DotNet%207.md#ref-to-ref-struct-argument-considered-unscoped-in-__arglist](https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20DotNet%207.md#ref-to-ref-struct-argument-considered-unscoped-in-__arglist).